src/plot_api/subroutines.js

```diff
@@ -466,12 +466,34 @@ exports.drawMainTitle = function(gd) {
 // supplyDefaults brought in an array that was already
 // in gd.data but not in gd._fullData previously
 exports.doTraceStyle = function(gd) {
-    for(var i = 0; i < gd.calcdata.length; i++) {
-        var cdi = gd.calcdata[i],
-            _module = ((cdi[0] || {}).trace || {})._module || {},
-            arraysToCalcdata = _module.arraysToCalcdata;
+    var fullLayout = gd._fullLayout;
+    var editStyleCalls = [];
+    var i;
+
+    for(i = 0; i < gd.calcdata.length; i++) {
+        var cd = gd.calcdata[i];
+        var cd0 = cd[0] || {};
+        var trace = cd0.trace || {};
+        var _module = trace._module || {};
+
+        var arraysToCalcdata = _module.arraysToCalcdata;
+        if(arraysToCalcdata) arraysToCalcdata(cd, trace);
+
+        var editStyle = _module.editStyle;
+        if(editStyle) editStyleCalls.push({fn: editStyle, cd0: cd0});
+    }
+
+    if(editStyleCalls.length) {
+        clearGlCanvases(gd);
```

Comment:
Is this going to cause problems if there's a `scattergl` or other regl trace along with `splom`? Do we need to implement `editStyle` for all of these at once?

Reply:
Good point. I might _have_ implement `editType: 'style'` for scattergl traces in this PR.

A better solution would be to implement https://github.com/plotly/plotly.js/issues/2562

Reply:
See https://github.com/plotly/plotly.js/pull/3067